src/Model/Table/QueuedTasksTable.php

```diff
@@ -67,11 +67,11 @@ public function createJob($jobName, $data = null, $notBefore = null, $group = nu
 		$data = [
 			'jobtype' => $jobName,
 			'data' => serialize($data),
-			'group' => $group,
+			'task_group' => $group,
 			'reference' => $reference
 		];
 		if ($notBefore !== null) {
-			$data['notbefore'] = date('Y-m-d H:i:s', strtotime($notBefore));
+			$data['notbefore'] =  strtotime($notBefore);
```

Comment:
The first days with CakePHP 3 made me some problems with saving formatet DATETIME fields when using German formating/language because of the different order 'd.m.Y H:i:s'. The ORM converts the timestamp automatical to an Time object when saving/retrieving.